tests/test_api_model_stubs.py

```diff
@@ -433,6 +516,51 @@ def test_returns_mapping_which_can_be_changed_using_kwargs(self, kwarg, key, val
         assert LotStub(**{kwarg: value}).response()[key] == value
 
 
+class TestServicesStubs:
```

Comment:
You can get rid of that `cls` fixture if you parametrize the class itself:
```
 
+@pytest.mark.parametrize('cls', [ArchivedServiceStub, DraftServiceStub, ServiceStub])
 class TestServicesStubs:
 
-    @pytest.fixture(params=[ArchivedServiceStub, DraftServiceStub, ServiceStub])
-    def cls(self, request):
-        return request.param
-
```

Reply:
I am just getting schooled today :laughing: Nice trick